packages/cfpb-layout/src/molecules/card.less

```diff
@@ -269,6 +269,11 @@
 
         // Arguments: default, `:visited`, `:hover`, `:focus`, and `:active` states.
         .u-link-card__colors( @pacific, @teal, @pacific-dark, @pacific, @navy );
+
+        // Breakout cards have larger links.
+        .m-card_footer > span {
+            .heading-3();
```

Comment:
We need to add margin or padding under this to make it equal-ish on the top and bottom of the text, ye?

Reply:
Mmmm yeah I posted a screenshot over in GHE but in the DS it definitely looks like it's running too close to the bottom border. I'll reference the comp and adjust spacing accordingly.